src/main/java/com/minecolonies/coremod/entity/ai/mobs/EntityArcherBarbarian.java

```diff
@@ -0,0 +1,85 @@
+package com.minecolonies.coremod.entity.ai.mobs;
+
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.coremod.sounds.BarbarianSounds;
+import net.minecraft.entity.projectile.EntityArrow;
+import net.minecraft.entity.projectile.EntitySpectralArrow;
+import net.minecraft.entity.projectile.EntityTippedArrow;
+import net.minecraft.init.Items;
+import net.minecraft.inventory.EntityEquipmentSlot;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.world.World;
+
+import javax.annotation.Nullable;
+
+/**
+ * Implements the Archer Barbarian Entity
+ */
+public class EntityArcherBarbarian extends AbstractArcherBarbarian
+{
+
+    /* default */ public static final ResourceLocation LOOT = new ResourceLocation(Constants.MOD_ID, "EntityArcherBarbarianDrops");
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AIndentationCheck)